src/studio/src/designer/frontend/packages/schema-editor/src/SchemaEditorApp.tsx

```diff
@@ -4,7 +4,7 @@ import './App.css';
 import SchemaEditor from './components/Editor';
 import { store } from './store';
 import type { ILanguage, ISchema } from './types';
-import { getTranslation } from './utils';
+import { getTranslation } from './utils/languageUtils';
```

Comment:
The `Utils` suffix is a bit redundant in these files under the `utils` directory.